includes/publisher/class-publisher-form-editing-experience.php

```diff
@@ -403,6 +404,47 @@ public function save_post( int $post_id, WP_Post $post, bool $update ): void {
 				}
 			}
 
+			$is_field_unique = get_field_from_slug( $slug, $this->models, $post->post_type )['isUnique'];
```

Comment:
This needs a fallback value in case the field does not have the isUnique property:

```suggestion
			$is_field_unique = get_field_from_slug( $slug, $this->models, $post->post_type )['isUnique'] ?? false;
```

Otherwise it produces errors on save for email fields that aren't unique:

<img width="1533" alt="Screenshot 2022-08-05 at 09 17 52" src="https://user-images.githubusercontent.com/647669/183023591-9e5a47ba-5f87-4813-97e2-39b655468541.png">

Reply:
I went ahead and fixed this in https://github.com/wpengine/atlas-content-modeler/pull/598/commits/007b2a81ec8fb27cd66ad0fa3bcbf83854dd9bdc. Hope that's ok!